Pod/Classes/PINProgressiveImage.h

```diff
@@ -10,15 +10,15 @@
 
 @interface PINProgressiveImage : NSObject
 
-@property (atomic, copy) NSArray *progressThresholds;
+@property (atomic, copy, nonnull) NSArray *progressThresholds;
 @property (atomic, assign) CFTimeInterval estimatedRemainingTimeThreshold;
 @property (atomic, assign) CFTimeInterval startTime;
 
-- (void)updateProgressiveImageWithData:(NSData *)data expectedNumberOfBytes:(int64_t)expectedNumberOfBytes;
+- (void)updateProgressiveImageWithData:(nonnull NSData *)data expectedNumberOfBytes:(int64_t)expectedNumberOfBytes;
 
 //Returns the latest image based on thresholds, returns nil if no new image is generated
-- (UIImage *)currentImage;
+- (nullable UIImage *)currentImage;
 
-- (NSData *)data;
+- (nonnull NSData *)data;
```

Comment:
This can actually be nil if called before updateProgressiveImageWithData is called. So either we need to return an NSData instance from the property if mutableData is nil, or mark this as nullable.